base/reducedim.jl

```diff
@@ -148,9 +148,9 @@ for (f1, f2, initval, typeextreme) in ((:min, :max, :Inf, :typemax), (:max, :min
             Tr = v0 isa T ? T : typeof(v0)
 
             # but NaNs and missing need to be avoided as initial values
-            if (v0 == v0) === false
+            if isnan(v0)
```

Comment:
How about `missing` (like the comment above discusses)?

Reply:
missing is handled below. (L153)

Reply:
Ah, ok.